### DISCUSSION TITLE
Cant find heated seats and steering wheel heater

### DISCUSSION THREAD

**Discussion by tonyvfield** [Hyundai-Kia-Connect/kia_uvo#799] [Q&A]
Hi, i am new to HA so appols if this question is silly.
I cant find the button to turn on or off the above two items. 
I am hoping to have a single button that turns on the AC, heated seats and steering wheel heater.
Sorry but since I am new please tell me exactly where to look to find these items to add to a card

**Comment by cdnninja** [comment]
Developer menu>services.   Search in that for Kia. 

It can be tied to a button or automation.

**Comment by tonyvfield** [comment]
No i dont have that if i search for swrvices. It does not appear

Regards
Tony Field

**Reply by cdnninja** [reply]
You don't search for services.  You browse to them in the dev menu and search within services.

**Comment by tonyvfield** [comment]
in that list there is no service to turn on the heated seats
![image](https://github.com/Hyundai-Kia-Connect/kia_uvo/assets/139866024/5810c3bd-e53e-4074-8d7d-9ccf15b1eb90)

**Reply by cdnninja** [reply]
Start_climate.  List is in the readme.

**Reply by tonyvfield** [reply]
Many thanks. I was looking for separate entities.
You are a star.
Now I just need to learn how to write a bit of code to have a button with options on my dash board
